source/guides/guides/parallelization.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Parallelization
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} What You'll Learn
+
+- 
+{% endnote %}
+
+# Overview
+
+If your project has a large number of tests, it can take a large amount of time for tests to complete running when run on one machine serially. Spreading your tests across many virtual machines and running tests in parallel can save your team time and money when running tests in Continuos Integration.
+
+Cypress can run recorded tests in parallel across multiple virtual machines.
+
+This guide assumes you already have your project running and recording within Continuous Integration. If you haven't set up your project yet, check out our Continuous Integration guide.
+
+# Splitting up your test suite
+
+Cypress' allocation strategy for parallelism is file-based, so in order to utilize parallelization, you will want to have your tests split across separate files.
+
+# Turning on parallelization
+
+You'll want to refer to your CI provider's documentation on how to set up multiple machines to run in your CI environment first.
+
+Once multiple machines are available within your CI environment, you can pass the {% url "`--parallel`" command-line#cypress-run-parallel %} key to {% url "`cypress run`" command-line#cypress-run %} to have your recorded tests parallelized.
+
+```shell
+cypress run --record --parallel
+```
+
+Running tests in parallel requires the `--record` flag be passed. This ensures Cypress can properly collect the data needed to parallelize the run. This also gives you the full benefit of seeing the results of the parallelization in our Dashboard Service. If you have not set up your project to record, check out our Set up guide.
```

Comment:
Consider hyperlinking to the set up guide to make navigating to the next thing easier.

Reply:
I suspect a PR for the documentation ain't the right place for this comment... but might Cypress default to turning reporting on when the parallel flag is set? Rationale: being explicit is often good but sensible defaulting is too.

Dunno: I'll watch this space, no biggie.

Reply:
We definitely do not want to record by default when the parallel flag is passed. 'recording' means that a select amount of your data is sent to our servers to deliver value in the Dashboard - and we'd hate for someone to think they are not sending us data when indeed they are.